src/Illuminate/Support/Uri.php

```diff
@@ -162,7 +162,7 @@ public function port(): ?int
      *
      * Empty or missing paths are returned as a single "/".
      */
-    public function path(): ?string
+    public function path(): string
```

Comment:
You can even go a step further

```suggestion
    * Empty or missing paths are returned as a single "/".
    * 
    * @return non-empty-string
    */
    public function path(): string
```